content/updates/2018-05-21-new-custom-domain-service.md

```diff
@@ -0,0 +1,9 @@
++++ 
+date = "2018-05-21" 
+title = "New custom domain service" 
++++
+
+### New custom domain service
+For custom domain support, cloud.gov provides a [CDN service](https://cloud.gov/docs/services/cdn-route/) that uses AWS CloudFront. AWS CloudFront is outside the AWS FedRAMP P-ATO boundary, so we’ve updated the CDN service documentation to [explain the compliance impact of using this service more clearly](http://localhost:1313/docs/services/cdn-route/#before-you-use-this-service).
```

Comment:
Needs real/non-localhost URL

Reply:
More info - we try to use this relative URL format in the docs, which makes it easier to keep the links working (you'll get a build error if a link is broken in a PR, so you know to fix it before merging and deploying the change) and also makes links work as expected when running the site locally: https://github.com/18F/cg-site#links